src/main/java/pl/pelikan/weatherinthecity/WeatherService.java

```diff
@@ -0,0 +1,25 @@
+package pl.pelikan.weatherinthecity;
+
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+import pl.pelikan.weatherinthecity.openweatherapi.OpenWeatherDto;
+
+@Service
+public class WeatherService {
+    private final String APIkey = "a679efea71b41c07773c5cefa663728b";
```

Comment:
APIkey bym bardziej widział w application.propertisach niż w kodzie. Wtedy jak będziemy musieli użyć innego apikey, to nie będzie problemu. Tylko przekażemy parametr podczas uruchamiania